core/crypto/sub.mk

```diff
@@ -1,4 +1,8 @@
 srcs-y += crypto.c
 srcs-y += aes-gcm.c
 srcs-y += aes-gcm-sw.c
+ifeq ($(CFG_AES_GCM_TABLE_BASED),y)
```

Comment:
`CFG_AES_GCM_TABLE_BASED` is not compatible with `CFG_CRYPTO_WITH_CE`. I think this should be checked somewhere. In `core/crypto.mk` maybe?

Reply:
You're right